rootston/keyboard.c

```diff
@@ -90,7 +101,35 @@ static bool keyboard_keysym_press(struct roots_keyboard *keyboard,
 		wlr_seat_keyboard_end_grab(keyboard->seat->seat);
 	}
 
-	uint32_t modifiers = wlr_keyboard_get_modifiers(keyboard->device->keyboard);
+	return false;
+}
+
+/**
+ * Press or release keysyms.
+ *
+ * Returns true if the keysym was handled by a binding and false if the event
+ * should be propagated to clients.
+ */
+static bool keyboard_handle_keysyms(struct roots_keyboard *keyboard,
+		xkb_keysym_t *pressed_keysyms, uint32_t modifiers,
+		const xkb_keysym_t *keysyms, size_t keysyms_len,
+		enum wlr_key_state state) {
```

Comment:
Could use a better name that indicates that it actually executes bindings like `keyboard_execute_bindings()`.